static/templates/bookend.handlebars

```diff
@@ -2,6 +2,13 @@
 
 {{#if bookend_content}}
 <div class="{{#if trailing}}trailing_bookend{{/if}} bookend sub-unsub-message">
+    {{^subscribed}}
     <span>{{bookend_content}}</span>
+    <div class="sub_button_row">
+        <button class="btn stream_sub_unsub_button btn-default" type="button" name="subscription">
+            Subscribe
+        </button>
```

Comment:
The subscribe buttons should probably be more happy and green; maybe it's worth including `btn-default btn-success` to the buttons inside the `sub_button_row` object here and in the other instance below?